src/interface/src/app/map/map.component.html

```diff
@@ -41,7 +41,7 @@
 
       <!-- Upload a planning area -->
       <div class="upload-wrapper">
-        <div matTooltip="This item is not available in this version." [matTooltipDisabled]="login_enabled">
+        <div matTooltip="This feature is not available at this time." [matTooltipDisabled]="login_enabled">
```

Comment:
This is a different change; ok to leave in, but please note the change in the PR description and title.

Reply:
Good point — fixed the title!